src/Button.tsx

```diff
@@ -33,6 +33,7 @@ export const buttonStyles = (theme: ITheme['button'], { disabled }: IButton = {}
 
       ':hover': {
         backgroundColor: theme.hoverBg,
+        ...(theme.hoverFg && { color: theme.hoverFg }),
```

Comment:
why need the `...` while background doesnt?

Reply:
It's optional

Reply:
Could also be
```color: theme.hoverFg || themefg```

Reply:
will css not figure it out if it gets an undefined value?

Reply:
@marbemac, it does, nice.
Code updated.